ironic-images/ipa-debian-bookworm/ipa-debian-bookworm.yaml

```
@@ -0,0 +1,8 @@
+- imagename: ipa-debian-bookworm
+  elements:
+  - ironic-python-agent-ramdisk
+  - debian
```

```suggestion
  - debian-minimal
```

Have we tried a smaller image?

---

I haven't tried debian-minimal yet.